src/main/java/com/rapid7/client/dcerpc/io/PacketInput.java

```diff
@@ -68,6 +68,33 @@ public int readReferentID()
         return result;
     }
 
+    /*
+       Similar to readString, except all that is offered by the byte stream is the number of bytes
+       String itself is null terminated
+     */
+    public String readChars()
```

Comment:
What is this method?  Doesn't seem like it belongs here (bad name)?

Reply:
Discussing this above.